deps.edn

```diff
@@ -1,4 +1,4 @@
-{:paths ["src/clj" "src/cljs" "src/cljc" "src/js" "src/gen" "test"]
+{:paths ["src/clj" "src/cljs" "src/cljc" "src/js" "src/gen" "test" "classes"]
```

Comment:
Why do we need compilation output to be treated as sources?
do we generate stuff that our code depends on?

Reply:
Added some changes that shouldn't need classes to be in the global paths, lmk if you have a better way.